src/components/ConversationView/index.tsx

```
@@ -0,0 +1,76 @@
+import ResponseLoader from "../../widgets/copilot/components/Loader";
+import SpinLoader from "src/components/shared/SpinLoader";
+import MessagesList from "./Responses";
+import { CopilotConfigType } from "src/contexts/types";
+import { OnSendCallbackType } from "src/widgets/copilot/components/ChatInput";
+
+export type ConversationViewProps = {
+  messages: Map<string, any>;
+  isSending?: boolean;
+  scrollContainerRef?: React.RefObject<HTMLDivElement> | null;
+  isMessagesLoading?: boolean;
+  avoidAutoplay?: () => void | null;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)